.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## Your Environment
+Thank you for taking the time to report an issue.
+
+To more efficiently resolve this issue, we'd like to know some basic information about your system and setup.
+
+1) Your OS (as much info as possible)
+* ie: Windows 10, macOS High Sierra version 10.13(17A365), Ubuntu 18.04 LTS, etc)
+
+2) Version of python you are running
+``` bash
+$ python --version
```

Comment:
In the past, I've seen folks become confused by a leading `$`. I'd like to avoid all confusion, if possible, and remove that from these invocations.